Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -2,244 +2,427 @@
-    return membershipID
+    return response_id
+
+
+def get_group(args, client, IdentityStoreId):
+    response_id = get_groupId_by_displayName(args, client, IdentityStoreId)
+    response = client.describe_group(
+        IdentityStoreId=f'{IdentityStoreId}',
+        GroupId=response_id.get('GroupId')
+    )
+    hr_data = {
+        'DisplayName': response.get('DisplayName'),
+        'GroupId': response['GroupId']
+    }
+    del response['ResponseMetadata']
+    ec = {'AWS.IAMIdentityCenter.Group(val.GroupId == obj.GroupId)': response}
+    human_readable = tableToMarkdown('AWS IAM Identity Center Groups', hr_data, removeNull=True)
+    result = CommandResults(
+        readable_output=human_readable,
+        outputs_key_field='GroupId',
+        outputs=ec
+    )
+    return_results(result)
 
 
-def add_user_to_group(args, client):  # pragma: no cover
-    userID = get_user(args, client)
-    GroupID = get_group(args, client)
+def list_groups_for_user(args, client, IdentityStoreId):
+    hr_data = []
+    context_data = {}
+    userID = get_userId_by_username(args, client, IdentityStoreId)['UserId']
+    kwargs = {
+        'IdentityStoreId': f'{IdentityStoreId}',
+        'MemberId': {
+            'UserId': f'{userID}'
+        },
+        'MaxResults': arg_to_number(args.get('limit') or params.get('limit')),
+        'NextToken': args.get('nextToken')
+    }
+    kwargs = remove_empty_elements(kwargs)
+    response = client.list_group_memberships_for_member(**kwargs)
+    groups = []
+    for group in response.get('GroupMemberships'):
+        hr_data.append({
+            'UserID': group['MemberId']['UserId'],
+            'GroupID': group['GroupId'],
+            'MembershipID': group['MembershipId']
+        })
+        groups.append({
+            'GroupId': group['GroupId'],
+            'MembershipId': group['MembershipId']
+        })
+    context_data['UserId'] = userID
+    context_data['GroupMemberships'] = groups
+    outputs = {'AWS.IAMIdentityCenter.User(val.UserId === obj.UserId)': context_data,
+               'AWS.IAMIdentityCenter(true)': {'GroupsUserNextToken': response.get('NextToken')}}
+    human_readable = tableToMarkdown('AWS IAM Identity Center Groups', hr_data, removeNull=True)
+    result = CommandResults(
+        readable_output=human_readable,
+        outputs_key_field='UserId',
+        outputs=outputs
+    )
+    return_results(result)
+
+
+def add_user_to_group(args, client, IdentityStoreId):
+    userID = get_userId_by_username(args, client, IdentityStoreId)['UserId']
+    GroupID = get_groupId_by_displayName(args, client, IdentityStoreId)['GroupId']
     response = client.create_group_membership(
-        IdentityStoreId=f'{IDENTITYSTOREID}',
+        IdentityStoreId=f'{IdentityStoreId}',
         GroupId=f'{GroupID}',
         MemberId={
             'UserId': f'{userID}'
         }
     )
+    membershipId = response.get('MembershipId')
     if response['ResponseMetadata']['HTTPStatusCode'] == 200:
```

Comment:
Why this is the only funciton you check the status code? 
what happens if the statuscode is not 200? you return None?